arangod/Utils/DatabaseGuard.cpp

```diff
@@ -29,13 +29,13 @@ namespace {
 
 template<typename T>
 TRI_vocbase_t& vocbase(arangodb::DatabaseFeature& feature, T& id) {
```

Comment:
I don't really understand this function, although it was available before this PR:
it returns a reference to a `TRI_vocbase_t`, but to do so it increases the vocbase's reference count... And it doesn't even return a pointer.
Can we get rid of the entire function?

Reply:
Sure